source/agent.py

```diff
@@ -101,11 +101,11 @@ def remove_extra_lines(file_path):
 def main():
     """Main function to handle command-line arguments and generate changelog"""
     parser = argparse.ArgumentParser(description='Process some integers.')
-    parser.add_argument('-u', '--url', type=str, default='https://github.com/nginx/agent/releases/', help='URL to fetch content from')
+    parser.add_argument('-u', '--url', type=str, default='https://github.com/spencerugbo/changelog-test/releases/', help='URL to fetch content from')
```

Comment:
is the URL change intended?

Reply:
No, leftover from testing. It should be reverted now